ballsdex/packages/info/cog.py

```diff
@@ -167,3 +169,65 @@ async def help(self, interaction: discord.Interaction):
                 )
 
         await interaction.response.send_message(embed=embed)
+
+    @app_commands.command()
+    async def tutorial(self, interaction: discord.Interaction):
+        """
+        View a descriptive tutorial of the bot.
+        """
+        trade_begin = Trade.begin.extras.get("mention", "/trade begin")
+        trade_add = Trade.add.extras.get("mention", "/trade add")
+        config_channel = Config.channel.extras.get("mention", "/config channel")
+
+        embed = discord.Embed(
+            title=f"{settings.bot_name.title()} Tutorial", color=discord.Color.blurple()
+        )
+
+        embed.add_field(
+            name="What can I do with this bot?",
+            value=(
+                f"You can collect {settings.plural_collectible_name}, exchange them with friends "
+                "and build a big and strong collection!"
+            ),
+            inline=False,
+        )
+
+        embed.add_field(
+            name="How can I configure the bot?",
+            value=(
+                f"To enable the spawning of {settings.plural_collectible_name}, you "
+                "need to configure the bot. To do that, you need to run the command "
+                f"{config_channel}. You must have the `Manage Guild` permission in that server."
```

Comment:
I dont think we need to mention in the current server, so it'd be better this way instead.
```suggestion
                f"{config_channel}. You must have the ``Manage Guild`` "
                "permission to use this command."
```